src/mlpack/core/data/load_csv.hpp

```diff
@@ -12,35 +32,92 @@
 #ifndef MLPACK_CORE_DATA_LOAD_CSV_HPP
 #define MLPACK_CORE_DATA_LOAD_CSV_HPP
 
-#include <boost/spirit/include/qi.hpp>
-#include <boost/algorithm/string/trim.hpp>
-
-#include <mlpack/core.hpp>
 #include <mlpack/core/util/log.hpp>
-
 #include <set>
 #include <string>
 
+#include "string_algorithms.hpp"
 #include "extension.hpp"
 #include "format.hpp"
 #include "dataset_mapper.hpp"
+#include "types.hpp"
 
 namespace mlpack {
 namespace data {
 
 /**
- *Load the csv file.This class use boost::spirit
- *to implement the parser, please refer to following link
- *http://theboostcpplibraries.com/boost.spirit for quick review.
+ * Load the csv file.This class use boost::spirit
+ * to implement the parser, please refer to following link
+ * http://theboostcpplibraries.com/boost.spirit for quick review.
  */
 class LoadCSV
 {
  public:
+
+  char delim;
+  // Do nothing, just a place holder, to be removed later.
+  LoadCSV()
+  {
+  } 
+
```

Comment:
You can remove it now

Reply:
@shrit I think we need to keep this as when we are initializing `LoadCSV` object as `LoadCSV loader` then this is the one which will get used, right?

Reply:
Ah, I missed that, in this case, you need to add a comment inside the body of the function such as: `// Nothing to do here.`, and remove the above comment of the function